### DISCUSSION TITLE
Ongoing Async Task (Websocket connection)

### DISCUSSION THREAD

**Discussion by lee-brown** [VentuzTechnology/Ventuz.Extension.prerelease#31] [Q&A]
Hi,

I've been playing around with VX to see what we might be able to do with it and I see some value in having a Websocket node with some additional functionality wrapped around it. 

Therefore to start with I'm looking to see if the VX system is suitable for maintaining a Websocket connection and outputting the latest message as a string. I noticed the roadmap in the wiki says "async resource generator for long-tasks", does this mean there isn't a safe way to create and maintain a Websocket connection using VX?

**Comment by VentuzKarolStrozynski** [comment]
You should be able to already achieve this as you do not need to go through the Ventuz Resource Mgnt.
You just have to maintain your connections on your own.

Best Regards
Karol
